notebooks/curation_api/R/get_dataset_assets_R.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c5490d93",
+   "metadata": {},
+   "source": [
+    "# Fetch download links for a Dataset's available assets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74560383",
+   "metadata": {},
+   "source": [
+    "The script in this notebook retrieves download links for a Dataset's assets.\n",
+    "\n",
+    "Fetching a Dataset assets requires only the Collection id (and, for specification purposes, the Dataset identifier), and not an API key/access token."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "968f003d",
+   "metadata": {},
+   "source": [
+    "#### <font color='#bc00b0'>Please fill in the required values:</font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0268c0a4",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Enter the id of the Collection that contains the Dataset for which you want to fetch download links</font>\n",
+    "\n",
+    "_The Collection id can be found by looking at the url path in the address bar \n",
+    "when viewing your Collection in the UI of the Data Portal website:_ `collections/{collection_id}`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc99f9db",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "collection_id <- \"01234567-89ab-cdef-0123-456789abcdef\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d7371ef",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Enter the identifier (curator tag or Dataset id) of the Dataset whose curator tag you want to update</font>\n",
+    "\n",
+    "_The `dataset_id` and `curator_tag` can both be found using the_ `/collections/{collection_id}` _endpoint and filtering for these attributes on the Dataset of interest._"
```

Comment:
Those `_` don't seem to be working if you look at the render.

Reply:
It renders in my pycharm environment. I will leave it as is and chalk it up to GitHub's problem.